lists/ke.csv

```diff
@@ -133,3 +133,7 @@ http://www.worldpoliticsreview.com/articles/17983/kenya-tries-to-stifle-any-crit
 http://www.wrapkenya.or.ke,HUMR,Human Rights Issues,2014-04-15,citizenlab,
 http://sportpesa.com/,GMB,Gambling,2017-03-20,CIPIT,
 https://sportpesa.com/,GMB,Gambling,2017-03-20,CIPIT,
+https://www.iebc.or.ke/,GOVT,Government,2017-05-25,CIPIT,
+https://jubileepamoja.co.ke/,MISC,Miscelaneous content,2017-05-25,CIPIT,
+https://rao.co.ke/,POLR,Political Criticism,2015-05-25,CIPIT,
+https://www.standardmedia.co.ke/article/2000229727/communications-authority-to-monitor-private-talk-and-texts-during-poll,NEWS,News Media,2017-05-25,CIPIT,

```

Comment:
As noted earlier, it would be interesting to see if the media company will revive the url.